deploy/docker-deploy.md

```
@@ -39,10 +53,11 @@ cp .env.example .env.project1
 cp .env.example .env.project2
 ```
 
-### Step 2: Edit each env file with unique ports
+### Step 2: Give each env file its own project name and ports
 
 **.env.project1** (default ports):
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)